exporter/awsemfexporter/emf_exporter.go

```diff
@@ -200,6 +204,14 @@ func (emf *emfExporter) start(_ context.Context, host component.Host) error {
 		return err
 	}
 
+	var userAgentExtras []string
```

Comment:
nit: we can remove some unused code around WithEnabledContainerInsights, WithEnabledAppSignals, UserAgentOption and UserAgentFlag in cwlog_client.go